docs/legacy/getting-started-apm-server.asciidoc

```diff
@@ -1,60 +1,175 @@
 [[getting-started-apm-server]]
-== Getting started with APM Server
+== Get started with APM Server (self managed)
 
 ++++
-<titleabbrev>Get started</titleabbrev>
+<titleabbrev>Get started (Self managed)</titleabbrev>
 ++++
 
+TIP: The easiest way to get started with Elastic APM is by using our
+{ess-product}[hosted {es} Service] on {ecloud}.
+The {es} Service is available on AWS, GCP, and Azure.
+See <<apm-quick-start>> to get started in minutes.
+
+// TODO: MOVE THIS
 IMPORTANT: Starting in version 8.0.0, {fleet} uses the APM integration to set up and manage APM index templates,
 {ilm-init} policies, and ingest pipelines. APM Server will only send data to {es} _after_ the APM integration has been installed.
 
-The easiest way to get started with Elastic APM is by using our
-{ess-product}[hosted {es} Service] on {ecloud}.
-The {es} Service is available on AWS, GCP, and Azure,
-and automatically configures APM Server to work with {es} and {kib}.
+The APM Server binary receives performance data from your APM agents,
```

Comment:
```suggestion
The APM Server receives performance data from your APM agents,
```

Reply:
Let's remove `binary` from here, as this is the case in both deployment options, and might be confusing otherwise.